features/cart/shopping_cart/clearing_cart_after_logging_out.feature

```diff
@@ -7,16 +7,27 @@ Feature: Clearing cart after logging out
     Background:
         Given the store operates on a single channel in "United States"
         And the store has a product "Stark T-shirt" priced at "$12.00"
-        And I am a logged in customer
-        And I have product "Stark T-Shirt" in the cart
 
     @ui
     Scenario: Clearing cart after logging out
+        Given I am a logged in customer
+        And I have product "Stark T-Shirt" in the cart
         When I log out
         And I see the summary of my cart
         Then my cart should be empty
 
     @api
     Scenario: Clearing cart after logging out
+        Given I am a logged in customer
+        And I have product "Stark T-Shirt" in the cart
         When I log out
-        Then I don't have access to see the summary of my previous cart
+        Then I should not have access to the summary of my previous cart
+
+    @api
+    Scenario: Blocking access to cart if logged user did any action over it (what can be treated as signing it)
```

Comment:
I have some doubts about the name of this scenario and about the feature file it is in, but I have also no better idea. Maybe we should move scenarios for API to a separate feature file, as it is not exactly clearing the cart, but blocking access to the previous cart? 🤔

Reply:
I'm open to rework it, but perhaps in different PR to move focus only to this problem?